zproject/prod_settings_template.py

```diff
@@ -273,12 +273,13 @@
 ## "APIs & Services" > "Credentials", and create a "Project" which will
 ## correspond to your Zulip instance.
 ##
-## (2) Navigate to "APIs & services" > "Library", and find the
-## "Identity Toolkit API".  Choose "Enable".
+## (2) Go to "Oauth consent screen" and create a consent screen,
+## authorizing your domain and enabling the .../auth/userinfo.email,
+## .../auth/userinfo.profile and openid scopes.
 ##
 ## (3) Return to "Credentials", and select "Create credentials".
-## Choose "OAuth client ID", and follow prompts to create a consent
-## screen.  Fill in "Authorized redirect URIs" with a value like
+## Choose "OAuth client ID", and follow prompts.
+## Fill in "Authorized redirect URIs" with a value like
 ##   https://zulip.example.com/complete/google/
 ## based on your value for EXTERNAL_HOST.
 ##
```

Comment:
Do we want to actually move all this stuff to `authentication-methods.md`? I recall we wanted to shift away from writing instructions in `prod_settings_template` and just move the to ReadTheDocs

Reply:
Yeah, I think we should take the opportunity to do that, since it's easily to update that way.

Reply:
Okay, makes sense.